pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -136,5 +135,29 @@ buildDotnetModule rec {
       MattSturgeon
     ];
     platforms = lib.platforms.linux;
+    description = "Game mod installer, creator and manager";
+    longDescription = ''
+      ${meta.description}.
+
+      ${
+        if _7zz.meta.unfree then
+          ''
+            This "unfree" variant includes support for mods packaged as RAR archives.
+          ''
+        else
+          ''
+            It is strongly recommended that you use the "unfree" variant of this package,
+            which provides support for mods packaged as RAR archives.
+
+            You can also enable unrar support manually, by overriding the `_7zz` used:
+
+            ```nix
+            pkgs.nexusmods-app.override {
+              _7zz = pkgs._7zz-rar;
+            }
+            ```
+          ''
+      }
+    '';
```

Comment:
The descriptions should probably be at or near the top of the `meta`. Most packages seem to have this crucial information near the top A quick scan seems to confirm this.

Reply:
I wanted `longDescription` near the bottom because it is, well, kinda long. So I ended up moving both descriptions.